src/wpool.erl

```diff
@@ -48,6 +48,10 @@
 -export([start_pool/1, start_pool/2, start_sup_pool/1, start_sup_pool/2]).
 -export([stop_pool/1]).
 -export([call/2, cast/2, call/3, cast/3, call/4, call/5]).
+-export([send_event/2, send_event/3, sync_send_event/2, sync_send_event/3,
+  sync_send_event/4, sync_send_event/5, send_all_event/2, send_all_event/3,
+  sync_send_all_event/2, sync_send_all_event/3, sync_send_all_event/4,
```

Comment:
Why do you rename some functions and not others?
I would either go the whole way and rename all the functions like [`ktn_fsm`](https://github.com/inaka/erlang-katana/blob/master/src/ktn_fsm.erl) does or keep the original names, like `sync_send_all_state_event`, for instance.